.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Reference
+
+[TALENT-X](https://orglab.atlassian.net/browse/TALENT-X)
```

Comment:
We probably don't need the `TALENT` part for this project and the link to JIRA there is probably specific to another project.

Reply:
Oh, ha! Good catch 👍